defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+yarn_version: "0.21.*"
+yarn_nodejs_version: "6.10.1"
```

Comment:
I don't think it's good to have a separate `yarn_nodejs_version` variable different from the `azavea.nodejs` variable of `nodejs_version`

This could lead to a project that already includes the `azavea.nodejs` role and specifies a `nodejs_version` to unexpectedly receive the wrong version of `Node` when adding this role and not specifying `yarn_nodejs_version`.

I would instead defer to the default in `azavea.nodejs` and allow users of this role to override the version with the variable used by that role already (`nodejs_version`).